src/Auth.ts

```diff
@@ -156,6 +213,13 @@ export class Auth {
 
     try {
       const service: Service = await this.getServiceConnectionInfo<Service>();
+
+      // 20231022: Graceful fallback for pre-multiaccount sign-ins
```

Comment:
Since this is a one-time action that is fixed with logging in again, let's not introduce code for it